torch/csrc/jit/tensorexpr/external_functions.cpp

```diff
@@ -467,12 +467,13 @@ void nnc_aten_mean(
 
   at::Tensor& r = tensors[0];
   const at::Tensor& x = tensors[1];
-  std::vector<int64_t> mean_dims(args_num);
+  std::vector<int64_t> mean_dims(args_num - 1);
+  bool keepdim = (bool)extra_args[args_num - 1];
   if (args_num > 0) {
-    memcpy(mean_dims.data(), extra_args, sizeof(int64_t) * args_num);
+    memcpy(mean_dims.data(), extra_args, sizeof(int64_t) * (args_num - 1));
```

Comment:
This allows 0 sized `memcpy`. Can that happen here? If not, can we guard against it?

Reply:
It shouldnt be possible (but also we can't really check it since we don't have error reporting mechanism yet).